src/pages/dashboard/components/navbar/NavBar.tsx

```diff
@@ -0,0 +1,498 @@
+import { NavLink } from 'react-router-dom'
+import './NavBar.scss'
+
+interface NavBarProps {
+  display: string
+}
+export const NavBar = ({ display }: NavBarProps) => {
+  return (
+    <>
+      <nav className="navbar" style={{ display: display }}>
+        <ul className="navbar-nav">
+          <li className="nav-item">
+            <div className="nav-link">
+              <NavLink to="/" className="link-text flex items-center">
+                <div>
+                  <svg
```

Comment:
svg image can be moved to a separate .svg file and import as an image here

Reply:
done